makefile.shade

```diff
@@ -21,66 +21,6 @@ var AUTHORS='Microsoft'
 use-standard-lifecycle
 k-standard-goals
 
-#xunit-test .xunit-test-msbuild
-
-#repack-x86 target='compile' if='Directory.Exists("src")'
```

Comment:
Were you able to verify that mult-targetting actually produces an x86 executable for net451? I haven't checked if that works.